mixer/cmd/root.go

```diff
@@ -33,12 +33,15 @@ import (
 
 var configFile string
 
+const containerMarker = "container"
+
 // RootCmd represents the base command when called without any subcommands
 var RootCmd = &cobra.Command{
 	Use:  "mixer",
 	Long: `Mixer is a tool used to compose OS update content and images.`,
 
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		//CPU Profiling
```

Comment:
For consistency with the rest of the codebase can you add a space after the `//` please?